### PR TITLE
Feature/add images

### PR DESCRIPTION
**Pull request by Imperssonator**
- added image of material history "twig" from GEMD slides to high-level-overview
- added diagrams of Attribute Templates and Object Templates to their respective pages in specification/
- added diagrams of "common motifs" (multiple output materials, multiple mixing steps) at the end of specification/objects.md

### CODE REVIEW

File: docs/high-level-overview.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ Object      | Examples   | Linked Objects | State | Possible Attributes
 
 *Table1:* Shows the relation of Objects to other Objects and Attributes possible in each state.
 
+![Core Objects](img/core-objects.png)
```
Comment (maxhutch) [inline docs/high-level-overview.md:54]: This image as the "Confidential" footer in it.  I assume it is not confidential, and would suggest cropping it.

File: docs/specification/attribute-templates.md [hunk 2]
```diff
@@ -10,6 +10,9 @@ with an Attribute Template must conform to the bounds specified by the template.
 If the value is not compatible with the template (e.g. if the template specifies that
 the value must be in the range [0, 1] but the value is 2), then the data are invalid.
 
+![attribute templates](img/attribute-templates.png)
```
Comment (maxhutch) [inline docs/specification/attribute-templates.md:13]: This image as the "Confidential" footer in it.  I assume it is not confidential, and would suggest cropping it.

File: docs/specification/attribute-templates.md [hunk 3]
```diff
@@ -22,6 +25,11 @@ Field name    | Value type | Default | Description
 `description` | String | None | Some text describing what this template is
 `tags`        | Set[String]| Empty set | [Tags](../tags)
 
+Attribute Templates are critical in defining which quantities are comparable or equivalent.
+When Attributes are added to core objects (Runs or Specs), they should be associated with
+an appropriate Attribute Template. This also determines how Values are collected and aggregated
```
Comment (maxhutch) [inline docs/specification/attribute-templates.md:30]:
```suggestion
an appropriate Attribute Template.
This association is meant to inform how Values are collected and aggregated
```
Newline added per sentence (markdown ignores single newlines just like latex).

File: docs/specification/attribute-templates.md [hunk 4]
```diff
@@ -22,6 +25,11 @@ Field name    | Value type | Default | Description
...
+in Tables.
```
Comment (maxhutch) [inline docs/specification/attribute-templates.md:31]:
```suggestion
in downstream consumers of GEMD data.
```
More generic.

File: docs/specification/object-templates.md [hunk 5]
```diff
@@ -17,6 +17,9 @@ but a Process Template describing a specific oven model may constrain the baking
 The Bounds must fall within the Attribute Template's Bounds, and should be set to those Bounds if no further constraint is desired.
 Each Attribute Template can only be included in a given Object Template's Attribute list once (e.g., once in `properties` and once in `conditions`).
 
+![All templates](img/templates.png)
```
Comment (maxhutch) [inline docs/specification/object-templates.md:20]: This image as the "Confidential" footer in it.  I assume it is not confidential, and would suggest cropping it.

File: docs/specification/objects.md [hunk 6]
```diff
@@ -762,3 +762,23 @@ Field name    | Value type | Default | Description
     "performed_date": "2015-03-14T15:09:27"
 }
 ```
+
+
+## Common Motifs
```
Comment (maxhutch) [inline docs/specification/objects.md:767]: Maybe this section should be included in the FAQ instead?

File: docs/specification/objects.md [hunk 7]
```diff
@@ -762,3 +762,23 @@ Field name    | Value type | Default | Description
...
+in the same piece of equipment.
+
+![Multi-output process](img/multiple-outputs.png)
+
+### Multiple Mixing Steps
+
+If a mixing process is run multiple times, for example, to incorporate "a little extra"
+of a certain ingredient, each subsequent mixing step should have a unique instance
+of an "Additional Mixing" Process Run and Spec, which, crucially, should point to a Process Template
+distinct from the initial mixing Process Template. This avoids the creation of a cycle.
```
Comment (maxhutch) [inline docs/specification/objects.md:782]: Is it a cycle or a ambiguous template association?  Another approach that @kroenlein, @lkubie , and I discussed was adding a parameter with a categorical value like "Initial" and "Additional" or "First", "Second", and "Third".

File: docs/specification/objects.md [hunk 8]
```diff
@@ -762,3 +762,23 @@ Field name    | Value type | Default | Description
...
+
+## Common Motifs
+
+### Multiple Outputs
+
+A process that produces many output Materials should be characterized through
+individual Process Runs for each output Material, even if the process occurred
+in the same piece of equipment.
+
+![Multi-output process](img/multiple-outputs.png)
```
Comment (maxhutch) [inline docs/specification/objects.md:775]: This image as the "Confidential" footer in it.  I assume it is not confidential, and would suggest cropping it.

File: docs/specification/objects.md [hunk 9]
```diff
@@ -762,3 +762,23 @@ Field name    | Value type | Default | Description
...
+
+### Multiple Mixing Steps
+
+If a mixing process is run multiple times, for example, to incorporate "a little extra"
+of a certain ingredient, each subsequent mixing step should have a unique instance
+of an "Additional Mixing" Process Run and Spec, which, crucially, should point to a Process Template
+distinct from the initial mixing Process Template. This avoids the creation of a cycle.
+
+![Multiple Mixing](img/multiple-mixing.png)
```
Comment (maxhutch) [inline docs/specification/objects.md:784]: This image as the "Confidential" footer in it.  I assume it is not confidential, and would suggest cropping it.

**Review by maxhutch — DISMISSED** [review]
The visuals are excellent, but they require some cropping or re-rendering to avoid the footer.  I've made some comments inline as well.

**Comment by sparadiso** [comment]
@kroenlein or @maxhutch is someone picking this up or should we close?

**Comment by kroenlein** [comment]
I'm supposed to be picking this up from Nils.

**Comment by djack** [comment]
Cleaning up my review queue. Cropped the images. The remaining inline comments don't seem like blockers to me.

**Review by djack — APPROVED** [review]

File: docs/specification/objects.md [hunk 10]
```diff
@@ -835,3 +835,23 @@ Field name    | Value type | Default | Description
...
+in the same piece of equipment.
+
+![Multi-output process](img/multiple-outputs.png)
+
+### Multiple Mixing Steps
+
+If a mixing process is run multiple times, for example, to incorporate "a little extra"
+of a certain ingredient, each subsequent mixing step should have a unique instance
+of an "Additional Mixing" Process Run and Spec, which, crucially, should point to a Process Template
+distinct from the initial mixing Process Template. This avoids the creation of a cycle.
```
Comment (maxhutch) [inline docs/specification/objects.md:855]:
```suggestion
distinct from the initial mixing Process Template.
```

**Pull request merged**